libs/payments/ui/src/lib/client/components/StripeWrapper.tsx

```diff
@@ -22,11 +28,36 @@ export function StripeWrapper({ amount, currency }: StripeWrapperProps) {
     amount,
     currency,
     paymentMethodCreation: 'manual',
+    appearance: {
+      variables: {
+        fontFamily:
+          'system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif',
+        fontSizeBase: '16px',
+        fontWeightNormal: '500',
+      },
+      rules: {
+        '.Tab': {
+          borderColor: 'rgba(0,0,0,0.3)',
+        },
+        '.Input': {
+          borderColor: 'rgba(0,0,0,0.3)',
+        },
+        '.Input::placeholder': {
+          color: '#767676',
+          fontWeight: '400',
+        },
+        '.Label': {
+          color: '#6D6D6E', // Based on grey-400 of tailwind.config.js
```

Comment:
~~Per https://tailwindcss.com/docs/customizing-colors#default-color-palette, it looks like `gray-400` is `#9ca3af`.~~

Nevermind, you said we're overriding the default colors in https://github.com/mozilla/fxa/blob/ce46102c869a41ed7d0e14320c95084da2dc54de/libs/shared/assets/src/configs/tailwind.config.js#L168